boards/Makefile.common

```diff
@@ -127,8 +134,14 @@ SIZE      ?= $(TOOLCHAIN)-size
 OBJCOPY   ?= $(TOOLCHAIN)-objcopy
 OBJDUMP   ?= $(TOOLCHAIN)-objdump
 
+# Set default flags to produce binary from .elf
+OBJCOPY_FLAGS ?= --output-target=binary
+# Add default flags to cargo. Boards can add additional options
+CARGO_FLAGS += $(VERBOSE) --target=$(TARGET) --package $(PLATFORM)  --target-dir=$(TARGET_DIRECTORY)
 # Set the default flags we need for objdump to get a .lst file.
-OBJDUMP_FLAGS += --disassemble-all --source --section-headers
+OBJDUMP_FLAGS ?= --disassemble-all --source --section-headers --demangle
+# Set default flags for size
+SIZE_FLAGS ?= 
```

Comment:
There are flags to size? Do we ever set these?

Reply:
It's not used in existing boards yet and I didn't set it not to confuse others. I use it with SIZE_FLAGS=-A,  to display more detailed information of section sizes. For the board I work on (risc-v based), i made linker script with dedicated sections .rodata and .rostrings to track easier where size difference come from:

    /* Read-only data sections. Separate it from .text to track size */
    .rostrings : 
    {
       /*
        * Strings all in one place so it's easier to count how much
        * space they waste.
        */
        __rodata_strings_start = .;
        *(.rodata.str1.* .rodata.str.* .rodata..L__unnamed* .rodata..Lanon.*)
        __rodata_strings_end = .;
    } > rom =0xffffffff
    .rodata :
    {
        *(.srodata* .rodata .rodata.* .gnu.linkonce.r.*)

       /*
        * Array of functions to call at init.  We only call these
        * when dumping coverage, so they'd better only be coverage
        * functions.
        */
        . = ALIGN(4);
	__coverage_funcs_start = .;
        KEEP (*(.init_array))
	__coverage_funcs_start = .;

        /*
         * Put coverage structures in ROM.  Unlike the default gcov
         * library, we don't modify them, so we don't need to spend
         * RAM on them.
         */
        KEEP(*(.sdata.__gcov_*))
        KEEP(*(.data.__gcov_*))

        /*
         * Explicitly put an empty word at the end of .rodata, so that
         * printing a string from .rodata will always terminate before
         * running off into data.
         */
        . = ALIGN(4);
        LONG(0);
    } > rom =0xffffffff